src/sdk/model/org/org.ts

```diff
@@ -1800,4 +1800,16 @@ export class Org extends Entity {
     });
   }
 
+  /**
+   * Retrieve a CSV report email with all event history for a given Org.
+   * @returns {Promise<Task>} task promise
+   */
+  /* istanbul ignore next: autogenerated */
+  async emailEventHistory(email: string): Promise<Task> {
```

Comment:
did you autogenerate these @JoMas971 ?

Reply:
no indeed... I've just copy paste it... Will need to write the tests then.

Reply:
i ask because the signature is wrong, these endpoints don't return tasks

Reply:
yeah... I just saw this... The vm signature was already there, so i just copy pasted it on other entity but i didn't see that the signature was wrong. I'm fixing it and add some tests ;-).

Reply:
why not just actually generate them using the plugin?